cytoscape-klayjs.js

```diff
@@ -0,0 +1,372 @@
+          var optionsOut = {};
+          var i = 0;
+          for(i in options){
+            if(options[i] !== 'undefined'){
+              var j = 0;
+              for(j in temp){
+                if(j === i){
+                  optionsOut[temp[j]] = options[i];
+                }
+              }
+            }
+          }
+          return optionsOut;
+        };
+
+        var children = function(){
+          var temp = [];
+          for(var i = 0; i < nodes.length; i++){
+            if(nodes[i].data().parent === undefined){
+              temp.push({
+                "id": nodes[i].data().id,
+                "width": 4,
+                "height": 4,
+                "children": [],
+              });
+            }else{
+              var j;
+              for(j = 0; j < temp.length; j++){
+                if(temp[j].id === nodes[i].data().parent){
+                  temp[j].children.push({
+                    "id": nodes[i].data().id,
+                    "width": 4,
+                    "height": 4,
+                    "children": []
+                  });
+                }
+              }
+            }
+          }
+          return temp;
+        };
+
+        var kedges = function(){
+          var temp = [];
```

Comment:
Usually functions like this will use a var like `ret` if they can't have a better name.  That way you at least know it's going to be returned.